hrjobcontract/views/modalForm.html

```diff
@@ -566,24 +566,27 @@ <h4 class="modal-title">{{copy.title}}</h4>
               </div>
               <div class="form-group" ng-repeat="leaveEntry in entity.leave">
                 <label for="{{prefix}}leave-{{leaveEntry.leave_type}}" class="col-sm-3 control-label">
-                {{options.leave.leave_type[leaveEntry.leave_type]}}
+                {{leaveEntry.leave_type_title}}
                 </label>
-                <div class="col-sm-3">
+                <div class="col-sm-2">
                   <input type="text" id="{{prefix}}leave-{{leaveEntry.leave_type}}"
                     name="leave-{{leaveEntry.leave_type}}"
                     class="form-control"
                     hrjc-number="2"
                     hrjc-number-float="true"
+                    hrjc-to-hours="leaveEntry.leave_calculation_unit_name === 'hours'"
```

Comment:
`hrjc-to-hours` and `hrjc-not-negative` looks like filters, please leave a note about them in the PR.

Reply:
Not filters but attrs to hrjc-number directive. Will add.